node/network/src/protocol.rs

```diff
@@ -118,6 +120,11 @@ impl<B: SignedBlock, Header: BlockHeader> Protocol<B, Header> {
         }
     }
 
+    pub fn get_node_by_account_id(&self, account_id: AccountId) -> Option<NodeIndex> {
+        let peer_account_info = self.peer_account_info.read();
+        Some(*peer_account_info.get(&account_id)?)
```

Comment:
Why not just `*peer_account_info.get(&account_id)`?

Reply:
it returns Option<&NodeIndex> and want to resolve that reference given it's usize.

Reply:
I see, how about `peer_account_info.get(&account_id).cloned()`?